de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/48-de.metas.inoutcandidate/5500120_sys_gh4507_M_ShipmentSchedule_GenerateShipment_QtyOptions_Process.sql

```diff
@@ -0,0 +1,105 @@
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(544223,'en_US') 
+;
+
+-- 2018-08-29T15:04:57.998
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para (AD_Client_ID,AD_Element_ID,AD_Org_ID,AD_Process_ID,AD_Process_Para_ID,AD_Reference_ID,ColumnName,Created,CreatedBy,EntityType,FieldLength,IsActive,IsAutocomplete,IsCentrallyMaintained,IsEncrypted,IsMandatory,IsRange,Name,SeqNo,Updated,UpdatedBy) VALUES (0,544223,0,540458,541336,17,'Quantity',TO_TIMESTAMP('2018-08-29 15:04:57','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.inoutcandidate',0,'Y','N','Y','N','N','N','Menge',40,TO_TIMESTAMP('2018-08-29 15:04:57','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2018-08-29T15:04:58.018
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para_Trl (AD_Language,AD_Process_Para_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Process_Para_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Process_Para t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Process_Para_ID=541336 AND NOT EXISTS (SELECT 1 FROM AD_Process_Para_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Process_Para_ID=t.AD_Process_Para_ID)
+;
+
+-- 2018-08-29T15:05:32.560
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference (AD_Client_ID,AD_Org_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,IsOrderByValue,Name,Updated,UpdatedBy,ValidationType) VALUES (0,0,540907,TO_TIMESTAMP('2018-08-29 15:05:32','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.inoutcandidate','Y','N','M_ShipmentSchedule_QtyToShip',TO_TIMESTAMP('2018-08-29 15:05:32','YYYY-MM-DD HH24:MI:SS'),100,'L')
+;
+
+-- 2018-08-29T15:05:32.570
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference_Trl (AD_Language,AD_Reference_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Reference_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Reference t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Reference_ID=540907 AND NOT EXISTS (SELECT 1 FROM AD_Reference_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Reference_ID=t.AD_Reference_ID)
+;
+
+-- 2018-08-29T15:06:10.817
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Ref_List_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541681,540907,TO_TIMESTAMP('2018-08-29 15:06:10','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.inoutcandidate','Y','Picked',TO_TIMESTAMP('2018-08-29 15:06:10','YYYY-MM-DD HH24:MI:SS'),100,'P','Picked')
```

Comment:
not sure if there is already some other agreement in place,
but i would like to have the german value in here. otherwise there will be another issue with more overhead later.
please get them from a german collegue, and also take care of the de_CH and en_US trls